recipes/embree/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+
+import os
+from conans import CMake, tools
+from conans.model.conan_file import ConanFile
+
+
+class Embree(ConanFile):
+    name = "embree"
+    version = "3.6.1"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "embree", "raytracing", "rendering")
+    description = "Intel's collection of high-performance ray tracing kernels."
+    generators = "cmake"
+    homepage = "https://embree.github.io/"
+    _source_folder = "source_subfolder"
+    _build_folder = "build_subfolder"
+    exports_sources = "CMakeLists.txt"
+    settings = "os", "compiler", "build_type", "arch"
+    requires = "tbb/2019_u9"
+
+    options = {
+        "fPIC": [
+            True,
+            False
+        ],
+        "backface_culling": [
+            True,
+            False
+        ],
+        "ignore_invalid_rays": [
+            True,
+            False
+        ],
+        "ray_masking": [
+            True,
+            False
+        ]
+    }
+
+    default_options = {"fPIC": True,
+                       "backface_culling": False,
+                       "ignore_invalid_rays": False,
+                       "ray_masking": False,
+                       "tbb:tbbmalloc": True,
```

Comment:
Is this option really required? The default TBB package does not offer pre-built package with this option.

Reply:
Yes I think embree requires it. client can specify "build missing" to build it from source. I tried it with the flag using gcc 5.5.0 and Apple Clang without any problem.